snapshot/archive.go

```diff
@@ -17,13 +17,13 @@ type ArchiveFormat = string
 
 const (
 	ArchiveTar     ArchiveFormat = "tar"
-	ArchiveTarball               = "tarball"
-	ArchiveZip                   = "zip"
+	ArchiveTarball ArchiveFormat = "tarball"
+	ArchiveZip     ArchiveFormat = "zip"
 )
 
 var (
 	ErrInvalidArchiveFormat = errors.New("unknown archive format")
-	ErrNotADirectory = errors.New("is not a directory")
+	ErrNotADirectory        = errors.New("is not a directory")
```

Comment:
actually this `ErrNotADirectory` is unused and can be removed -- I'll do it in a different PR to not clash with this one.

Reply:
ok, I'm closing this conversation then!